src/components/Footer/Footer.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <footer class="Footer">
+      <p class="FooterText">©2020 Anem per feina</p>
```

Comment:
In this case, it is better to have an extra container:

```html
<footer class="Footer">
  <div class="Content">
      <p class="FooterText">©2020 Anem per feina</p>
  </div>
</footer>
```

This way it will be easier to add more things to the footer and the spacing will be more consistent, let me explain better.

You have to take into account how `rem` and `em` units work, look at the following cases:

### Case 1
```css
body {
  font-size: 16px /*default browser value*/
}

.element {
  font-size: 1rem; /*equals 16px*/
  padding: 0.5em; /*equals 8px*/
}
```

### Case 2
```css
body {
  font-size: 8px
}

.element {
  font-size: 1rem; /*equals 8px*/
  padding: 0.5em; /*equals 4px*/
}
```

As you can see, the `r` in `rem` stands for "root" so it will use the root font size as a reference. On the other side, the `em` unit will use the current element font size.

Regarding your code,  this means that the padding of the Footer is computed in relation to the paragraph text, which is not very straight-forward. By using two containers you will be able to use the same padding that the Header is using, and apply styles that belong to the paragraph just to the paragraph.